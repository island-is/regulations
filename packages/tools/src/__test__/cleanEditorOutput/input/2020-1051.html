<h3 class="article__title">1. gr. <em class="article__name">Markmi&eth;.</em></h3>
<p>Me&eth; v&iacute;san til 2. mgr. 12. gr. s&oacute;ttvarnalaga nr. 19/1997 hefur heilbrig&eth;isr&aacute;&eth;herra &aacute;kve&eth;i&eth;, a&eth; feng&shy;inni till&ouml;gu s&oacute;ttvarnal&aelig;knis og &iacute; samr&aacute;&eth;i vi&eth; r&iacute;kisstj&oacute;rnina, a&eth; setja &aacute; t&iacute;mabundna takm&ouml;rkun &aacute; samkomum eftir &thorn;v&iacute; sem n&aacute;nar greinir &iacute; regluger&eth; &thorn;essari.</p>
<p>Markmi&eth; takm&ouml;rkunarinnar er a&eth; h&aelig;gja eins og unnt er &aacute; &uacute;tbrei&eth;slu COVID-19 sj&uacute;kd&oacute;msins.</p>
<h3 class="article__title">2. gr. <em class="article__name">Gildissvi&eth;.</em></h3>
<p>Takm&ouml;rkun &aacute; samkomum tekur gildi 31. okt&oacute;ber 2020 og gildir til og me&eth; 17. n&oacute;vember 2020.</p>
<p>Stj&oacute;rnv&ouml;ld endurmeta &thorn;&ouml;rf &aacute; takm&ouml;rkuninni eftir &thorn;v&iacute; sem efni standa til, &thorn;.e. hvort unnt s&eacute; a&eth; afl&eacute;tta henni fyrr e&eth;a hvort &thorn;&ouml;rf s&eacute; &aacute; a&eth; framlengja gildist&iacute;ma hennar.</p>
<p>Takm&ouml;rkun &aacute; samkomum tekur til landsins alls.</p>
<p>Regluger&eth;in tekur hvorki til al&thorn;j&oacute;&eth;aflugvalla og -hafna n&eacute; til loft&shy;fara og skipa &iacute; millilanda&shy;fer&eth;um.</p>
<p>&Aacute;kv&aelig;&eth;i 3. og 4. gr. regluger&eth;ar &thorn;essarar um fj&ouml;ldatakm&ouml;rkun, n&aacute;l&aelig;g&eth;artakm&ouml;rkun og gr&iacute;mu&shy;skyldu taka ekki til barna sem f&aelig;dd eru 2015 e&eth;a s&iacute;&eth;ar.</p>
<p>Regluger&eth; &thorn;essi tekur ekki til sk&oacute;lastarfs sem fjalla&eth; er um &iacute; s&eacute;rstakri regluger&eth; um takm&ouml;rkun sk&oacute;lastarfs.</p>
<h3 class="article__title">3. gr. <em class="article__name">Fj&ouml;ldatakm&ouml;rkun.</em></h3>
<p>&Iacute; takm&ouml;rkun &aacute; samkomum felst a&eth; fj&ouml;ldasamkomur eru &oacute;heimilar &aacute; gildist&iacute;ma &thorn;essarar reglu&shy;ger&eth;ar. Me&eth; fj&ouml;ldasamkomum er &aacute;tt vi&eth; &thorn;a&eth; &thorn;egar fleiri en t&iacute;u einstaklingar koma saman, hvort sem er &iacute; opinberum r&yacute;mum e&eth;a einkar&yacute;mum. Er &thorn;&aacute; me&eth;al annars v&iacute;sa&eth; til:</p>
<ol style="list-style-type: lower-alpha;">
<li>R&aacute;&eth;stefna, m&aacute;l&thorn;inga, &uacute;tifunda o.&thorn;.h.</li>
<li>Kennslu, fyrirlestra og pr&oacute;fahalds.</li>
<li>Skemmtana, svo sem t&oacute;nleika, menningarvi&eth;bur&eth;a, leiks&yacute;ninga, b&iacute;&oacute;s&yacute;ninga, &iacute;&thorn;r&oacute;ttavi&eth;bur&eth;a og einkasamkv&aelig;ma.</li>
<li>Kirkjuathafna hvers konar, svo sem vegna giftinga og ferminga, og annarra tr&uacute;ar&shy;samkoma, sbr. &thorn;&oacute; 4. mgr.</li>
<li>Annarra samb&aelig;rilegra vi&eth;bur&eth;a me&eth; fleiri en t&iacute;u einstaklingum.</li>
</ol>
<p>Enn fremur skal tryggt &aacute; &ouml;llum vinnust&ouml;&eth;um og &iacute; allri starfsemi a&eth; ekki s&eacute;u &aacute; sama t&iacute;ma fleiri en t&iacute;u einstaklingar inni &iacute; sama r&yacute;mi. Tryggja skal a&eth; ekki s&eacute; samgangur milli r&yacute;ma.</p>
<p>&THORN;r&aacute;tt fyrir 1. og 2. mgr. er lyfja- og matv&ouml;ruverslunum heimilt a&eth; hafa allt a&eth; 50 manns inni &iacute; einu a&eth; &thorn;v&iacute; gefnu a&eth; uppfyllt s&eacute;u skilyr&eth;i 4. gr. &THORN;&aacute; er lyfja- og matv&ouml;ruverslunum sem eru yfir 1.000 m&sup2; a&eth; st&aelig;r&eth; heimilt a&eth; hleypa til vi&eth;b&oacute;tar einum vi&eth;skiptavin inn fyrir hverja 10 m&sup2; umfram 1.000 m&sup2;, &thorn;&oacute; a&eth; h&aacute;marki 100 vi&eth;skipta&shy;vinum alls.</p>
<p>&THORN;r&aacute;tt fyrir 1. og 2. mgr. er allt a&eth; 30 einstaklingum heimilt a&eth; vera vi&eth;staddir &uacute;tfarir. Um fj&ouml;lda f&oacute;lks vi&eth; erfidrykkju fer eftir 1. mgr.</p>
<p>Fj&ouml;ldatakmarkanir 1. og 2. mgr. gilda ekki um almenningssamg&ouml;ngur, h&oacute;pbifrei&eth;ar, innan&shy;lands&shy;flug, far&thorn;egaferjur og st&ouml;rf vi&eth;brag&eth;sa&eth;ila, svo sem l&ouml;greglu, sl&ouml;kkvili&eth;s, hj&aacute;lparli&eth;s almanna&shy;varna og heilbrig&eth;isstarfsf&oacute;lks. Enn fremur gildir &aacute;kv&aelig;&eth;i&eth; ekki um st&ouml;rf r&iacute;kisstj&oacute;rnar, r&iacute;kisr&aacute;&eth;s, Al&thorn;ingis og &thorn;egar d&oacute;m&shy;st&oacute;lar fara me&eth; d&oacute;msvald sitt.</p>
<p>Fj&ouml;ldatakm&ouml;rkun samkv&aelig;mt &aacute;kv&aelig;&eth;i &thorn;essu &aacute; ekki vi&eth; um heimilisf&oacute;lk &aacute; heimili s&iacute;nu.</p>
<h3 class="article__title">4. gr. <em class="article__name">Almenn n&aacute;l&aelig;g&eth;artakm&ouml;rkun og gr&iacute;munotkun.</em></h3>
<p>&Aacute; samkomum, &ouml;llum vinnust&ouml;&eth;um og &iacute; allri annarri starfsemi, me&eth;al annars &thorn;eirri sem talin er upp &iacute; 3. gr., skal tryggja a&eth; h&aelig;gt s&eacute; a&eth; hafa a.m.k. 2 metra &aacute; milli einstaklinga sem ekki eru &iacute; n&aacute;num tengslum.</p>
<p>Andlitsgr&iacute;mur skal nota &iacute; almenningssamg&ouml;ngum, &iacute; verslunum og annarri &thorn;j&oacute;nustu. Einnig skal nota andlitsgr&iacute;mur &thorn;ar sem ekki er unnt a&eth; tryggja 2 metra n&aacute;l&aelig;g&eth;artakm&ouml;rkun sem kve&eth;i&eth; er &aacute; um &iacute; 1. mgr., svo sem &iacute; heilbrig&eth;is&thorn;j&oacute;nustu, leigubifrei&eth;um og h&oacute;pbifrei&eth;um. Andlitsgr&iacute;ma skal hylja nef og munn. Andlitsgr&iacute;mur sem nota&eth;ar eru utan heilbrig&eth;is&thorn;j&oacute;nustu skulu uppfylla kr&ouml;fur sem fram koma &iacute; vinnustofusam&thorn;ykkt evr&oacute;psku sta&eth;la&shy;samtakanna (CEN). S&oacute;ttvarnal&aelig;knir getur sett n&aacute;nari lei&eth;beiningar um kr&ouml;fur til andlitsgr&iacute;ma sam&shy;kv&aelig;mt &aacute;kv&aelig;&eth;i &thorn;essu.</p>
<h3 class="article__title">5. gr. <em class="article__name">Takmarkanir &aacute; starfsemi vegna s&eacute;rstakrar smith&aelig;ttu.</em></h3>
<p>Starfsemi og &thorn;j&oacute;nusta sem krefst snertingar e&eth;a h&aelig;tta er &aacute; snertingu milli f&oacute;lks e&eth;a mikillar n&aacute;l&aelig;g&eth;ar er &oacute;heimil, svo sem menningarstarfsemi, starfsemi h&aacute;rgrei&eth;slustofa, snyrtistofa, nudd&shy;stofa, h&uacute;&eth;fl&uacute;rstofa, hundasnyrtistofa, s&oacute;lba&eth;sstofa og &ouml;nnur samb&aelig;rileg starfsemi. Takmark&shy;anir &thorn;essar taka ekki til starfsemi heilbrig&eth;isstarfsmanna vi&eth; veitingu heilbrig&eth;is&thorn;j&oacute;nustu en &iacute; &thorn;eim tilvikum er skylt a&eth; notast vi&eth; andlitsgr&iacute;mur. &Aacute;kv&aelig;&eth;i &thorn;etta tekur ekki til innanlandsflugs og -ferja, leigubifrei&eth;a, h&oacute;p&shy;bifrei&eth;a og almenningssamgangna.</p>
<p>Skemmtist&ouml;&eth;um og kr&aacute;m, sbr. 18. gr. regluger&eth;ar nr. 1277/2016, um veitingasta&eth;i, gistista&eth;i og skemmtanahald, og spilas&ouml;lum skal loka&eth; fr&aacute; gildist&ouml;ku regluger&eth;arinnar. &THORN;&aacute; skulu a&eth;rir veitinga&shy;sta&eth;ir &thorn;ar sem heimila&eth;ar eru &aacute;fengisveitingar ekki hafa opi&eth; lengur en til kl. 21.00 alla daga vik&shy;unnar og fylgja &aacute;kv&aelig;&eth;um 3. og 4. gr. &THORN;&oacute; er heimilt a&eth; selja mat &uacute;t &uacute;r h&uacute;si til kl. 23.00.</p>
<p>&Iacute; &thorn;eim tilvikum &thorn;ar sem fleiri tegundir veitingasta&eth;a eru skr&aacute;&eth;ar &iacute; rekstrarleyfi er heimilt a&eth; starfa &aacute;fram &iacute; samr&aelig;mi vi&eth; a&eth;rar tegundir en kve&eth;i&eth; er &aacute; um &iacute; 2. mgr. me&eth; &thorn;eim takm&ouml;rkunum sem kve&eth;i&eth; er &aacute; um &iacute; regluger&eth; &thorn;essari.</p>
<p>Ekki er heimilt a&eth; veita t&iacute;mabundi&eth; leyfi fyrir skemmtunum, svo sem t&oacute;nleikum, dansleikjum, brennum e&eth;a &ouml;&eth;rum vi&eth;bur&eth;um sem &aelig;tla m&aacute; a&eth; dragi a&eth; s&eacute;r h&oacute;p f&oacute;lks eftir kl. 21.00.</p>
<p>Sund- og ba&eth;st&ouml;&eth;um sem og l&iacute;kamsr&aelig;ktarst&ouml;&eth;vum skal loka&eth;.</p>
<p>&Iacute;&thorn;r&oacute;ttir, &thorn;ar me&eth; tali&eth; &aelig;fingar og keppnir, barna og fullor&eth;inna hvort sem er innan- e&eth;a utan&shy;dyra, me&eth; e&eth;a &aacute;n snertingar eru &oacute;heimilar. Einstaklingsbundnar &aelig;fingar &aacute;n snertingar eru heimilar, svo sem &uacute;tihlaup og samb&aelig;rileg hreyfing.</p>
<p>Svi&eth;slistir, og samb&aelig;rileg starfsemi, eru &oacute;heimilar.</p>
<h3 class="article__title">6. gr. <em class="article__name">Hj&uacute;krunarheimili og a&eth;rar heilbrig&eth;isstofnanir.</em></h3>
<p>Heilbrig&eth;isstofnanir, hj&uacute;krunarheimili og a&eth;rar samb&aelig;rilegar stofnanir skulu setja s&eacute;r reglur um starfsemi s&iacute;na, svo sem var&eth;andi heims&oacute;knir utana&eth;komandi a&eth; heimilunum og stofnununum.</p>
<h3 class="article__title">7. gr. <em class="article__name">&THORN;rif og s&oacute;tthreinsun almenningsr&yacute;ma.</em></h3>
<p>&Iacute; &ouml;llum verslunum, opinberum byggingum og &aacute; &ouml;&eth;rum st&ouml;&eth;um innandyra &thorn;ar sem umgangur f&oacute;lks er nokkur skal &thorn;r&iacute;fa eins oft og unnt er, s&eacute;rstaklega algenga snertifleti. Auk &thorn;ess skal minna almenning og starfsmenn &aacute; einstaklingsbundnar s&oacute;ttvarnir, svo sem munnlega, me&eth; merkingum e&eth;a skiltum.</p>
<p>Vi&eth; alla innganga skal tryggja a&eth;gang a&eth; s&oacute;tthreinsandi v&ouml;kva fyrir hendur og eins v&iacute;&eth;a um r&yacute;mi og talin er &thorn;&ouml;rf &aacute;, me&eth;al annars vi&eth; afgrei&eth;slukassa og innkaupakerrur &iacute; verslunum.</p>
<h3 class="article__title">8. gr. <em class="article__name">Undan&thorn;&aacute;gur.</em></h3>
<p>R&aacute;&eth;herra getur veitt undan&thorn;&aacute;gu fr&aacute; takm&ouml;rkun &aacute; samkomum vegna samf&eacute;lagslega &oacute;missandi innvi&eth;a sem mega ekki st&ouml;&eth;vast, svo sem &aacute; svi&eth;i raforku, fjarskipta, samgangna, sorphir&eth;u, heil&shy;brig&eth;is&shy;starfsemi, f&eacute;lags&thorn;j&oacute;nustu, l&ouml;gg&aelig;slu, sj&uacute;kraflutninga, hj&aacute;lparli&eth;s almannavarna e&eth;a sl&ouml;kkvi&shy;li&eth;a.</p>
<p>Hi&eth; sama &aacute; vi&eth; vegna kerfislega og efnahagslega mikilv&aelig;grar starfsemi, svo sem til a&eth; tryggja f&aelig;&eth;u&ouml;ryggi, dreifingu nau&eth;synjavara e&eth;a efnahagslegt &ouml;ryggi r&iacute;kisins me&eth; &uacute;tflutningi. Enn fremur getur r&aacute;&eth;herra veitt undan&thorn;&aacute;gu vegna starfa Al&thorn;ingis, d&oacute;mst&oacute;la og stj&oacute;rns&yacute;slu.</p>
<p>R&aacute;&eth;herra getur veitt undan&thorn;&aacute;gu fr&aacute; banni vi&eth; &iacute;&thorn;r&oacute;ttastarfi fyrir einstaka vi&eth;bur&eth;i, svo sem vegna al&thorn;j&oacute;&eth;legra keppnisleikja h&eacute;r &aacute; landi, enda s&eacute; g&aelig;tt fyllstu s&oacute;ttvarnar&aacute;&eth;stafana a&eth; &ouml;&eth;ru leyti.</p>
<p>Vi&eth; veitingu undan&thorn;&aacute;ga skal leitast vi&eth; a&eth; gr&iacute;pa til annarra a&eth;ger&eth;a &thorn;annig a&eth; g&aelig;tt s&eacute; a&eth; smit&shy;v&ouml;rnum eins og kostur er.</p>
<h3 class="article__title">9. gr. <em class="article__name">Gildistaka.</em><span class="article__name">&nbsp;&nbsp;&nbsp; </span></h3>
<p>Regluger&eth; &thorn;essi, sem sett er me&eth; heimild &iacute; 12. gr. s&oacute;ttvarnalaga nr. 19/1997, tekur gildi eftir &thorn;v&iacute; sem m&aelig;lt er fyrir um &iacute; 2. gr. Fr&aacute; sama t&iacute;ma fellur &uacute;r gildi regluger&eth; nr. 1015/2020, um takm&ouml;rkun &aacute; samkomum vegna fars&oacute;ttar.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p class="Dags" style="text-align: center;"><em>Heilbrig&eth;isr&aacute;&eth;uneytinu, 30. okt&oacute;ber 2020.</em></p>
<p class="FHUndirskr" style="text-align: center;">F. h. r.</p>
<p style="text-align: center;"><strong>&Aacute;sta Valdimarsd&oacute;ttir.</strong></p>
<p style="text-align: right;"><em>&Aacute;sthildur Kn&uacute;tsd&oacute;ttir.</em></p>