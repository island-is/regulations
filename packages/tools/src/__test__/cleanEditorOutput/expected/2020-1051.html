<h3 class="article__title">
1.
gr.
<em class="article__name">Markmið.</em>
</h3>

<p>
Með
vísan
til
2.
mgr.
12.
gr.
sóttvarnalaga
nr.
19/1997
hefur
heilbrigðisráðherra
ákveðið,
að
fenginni
tillögu
sóttvarnalæknis
og
í
samráði
við
ríkisstjórnina,
að
setja
á
tímabundna
takmörkun
á
samkomum
eftir
því
sem
nánar
greinir
í
reglugerð
þessari.
</p>

<p>
Markmið
takmörkunarinnar
er
að
hægja
eins
og
unnt
er
á
útbreiðslu
COVID-19
sjúkdómsins.
</p>

<h3 class="article__title">
2.
gr.
<em class="article__name">Gildissvið.</em>
</h3>

<p>
Takmörkun
á
samkomum
tekur
gildi
31.
október
2020
og
gildir
til
og
með
17.
nóvember
2020.
</p>

<p>
Stjórnvöld
endurmeta
þörf
á
takmörkuninni
eftir
því
sem
efni
standa
til,
þ.e.
hvort
unnt
sé
að
aflétta
henni
fyrr
eða
hvort
þörf
sé
á
að
framlengja
gildistíma
hennar.
</p>

<p>
Takmörkun
á
samkomum
tekur
til
landsins
alls.
</p>

<p>
Reglugerðin
tekur
hvorki
til
alþjóðaflugvalla
og
-hafna
né
til
loftfara
og
skipa
í
millilandaferðum.
</p>

<p>
Ákvæði
3.
og
4.
gr.
reglugerðar
þessarar
um
fjöldatakmörkun,
nálægðartakmörkun
og
grímuskyldu
taka
ekki
til
barna
sem
fædd
eru
2015
eða
síðar.
</p>

<p>
Reglugerð
þessi
tekur
ekki
til
skólastarfs
sem
fjallað
er
um
í
sérstakri
reglugerð
um
takmörkun
skólastarfs.
</p>

<h3 class="article__title">
3.
gr.
<em class="article__name">Fjöldatakmörkun.</em>
</h3>

<p>
Í
takmörkun
á
samkomum
felst
að
fjöldasamkomur
eru
óheimilar
á
gildistíma
þessarar
reglugerðar.
Með
fjöldasamkomum
er
átt
við
það
þegar
fleiri
en
tíu
einstaklingar
koma
saman,
hvort
sem
er
í
opinberum
rýmum
eða
einkarýmum.
Er
þá
meðal
annars
vísað
til:
</p>

<ol type="a">

<li>
Ráðstefna,
málþinga,
útifunda
o.þ.h.
</li>

<li>
Kennslu,
fyrirlestra
og
prófahalds.
</li>

<li>
Skemmtana,
svo
sem
tónleika,
menningarviðburða,
leiksýninga,
bíósýninga,
íþróttaviðburða
og
einkasamkvæma.
</li>

<li>
Kirkjuathafna
hvers
konar,
svo
sem
vegna
giftinga
og
ferminga,
og
annarra
trúarsamkoma,
sbr.
þó
4.
mgr.
</li>

<li>
Annarra
sambærilegra
viðburða
með
fleiri
en
tíu
einstaklingum.
</li>

</ol>

<p>
Enn
fremur
skal
tryggt
á
öllum
vinnustöðum
og
í
allri
starfsemi
að
ekki
séu
á
sama
tíma
fleiri
en
tíu
einstaklingar
inni
í
sama
rými.
Tryggja
skal
að
ekki
sé
samgangur
milli
rýma.
</p>

<p>
Þrátt
fyrir
1.
og
2.
mgr.
er
lyfja-
og
matvöruverslunum
heimilt
að
hafa
allt
að
50
manns
inni
í
einu
að
því
gefnu
að
uppfyllt
séu
skilyrði
4.
gr.
Þá
er
lyfja-
og
matvöruverslunum
sem
eru
yfir
1.000
m²
að
stærð
heimilt
að
hleypa
til
viðbótar
einum
viðskiptavin
inn
fyrir
hverja
10
m²
umfram
1.000
m²,
þó
að
hámarki
100
viðskiptavinum
alls.
</p>

<p>
Þrátt
fyrir
1.
og
2.
mgr.
er
allt
að
30
einstaklingum
heimilt
að
vera
viðstaddir
útfarir.
Um
fjölda
fólks
við
erfidrykkju
fer
eftir
1.
mgr.
</p>

<p>
Fjöldatakmarkanir
1.
og
2.
mgr.
gilda
ekki
um
almenningssamgöngur,
hópbifreiðar,
innanlandsflug,
farþegaferjur
og
störf
viðbragðsaðila,
svo
sem
lögreglu,
slökkviliðs,
hjálparliðs
almannavarna
og
heilbrigðisstarfsfólks.
Enn
fremur
gildir
ákvæðið
ekki
um
störf
ríkisstjórnar,
ríkisráðs,
Alþingis
og
þegar
dómstólar
fara
með
dómsvald
sitt.
</p>

<p>
Fjöldatakmörkun
samkvæmt
ákvæði
þessu
á
ekki
við
um
heimilisfólk
á
heimili
sínu.
</p>

<h3 class="article__title">
4.
gr.
<em class="article__name">Almenn
nálægðartakmörkun
og
grímunotkun.</em>
</h3>

<p>
Á
samkomum,
öllum
vinnustöðum
og
í
allri
annarri
starfsemi,
meðal
annars
þeirri
sem
talin
er
upp
í
3.
gr.,
skal
tryggja
að
hægt
sé
að
hafa
a.m.k.
2
metra
á
milli
einstaklinga
sem
ekki
eru
í
nánum
tengslum.
</p>

<p>
Andlitsgrímur
skal
nota
í
almenningssamgöngum,
í
verslunum
og
annarri
þjónustu.
Einnig
skal
nota
andlitsgrímur
þar
sem
ekki
er
unnt
að
tryggja
2
metra
nálægðartakmörkun
sem
kveðið
er
á
um
í
1.
mgr.,
svo
sem
í
heilbrigðisþjónustu,
leigubifreiðum
og
hópbifreiðum.
Andlitsgríma
skal
hylja
nef
og
munn.
Andlitsgrímur
sem
notaðar
eru
utan
heilbrigðisþjónustu
skulu
uppfylla
kröfur
sem
fram
koma
í
vinnustofusamþykkt
evrópsku
staðlasamtakanna
(CEN).
Sóttvarnalæknir
getur
sett
nánari
leiðbeiningar
um
kröfur
til
andlitsgríma
samkvæmt
ákvæði
þessu.
</p>

<h3 class="article__title">
5.
gr.
<em class="article__name">Takmarkanir
á
starfsemi
vegna
sérstakrar
smithættu.</em>
</h3>

<p>
Starfsemi
og
þjónusta
sem
krefst
snertingar
eða
hætta
er
á
snertingu
milli
fólks
eða
mikillar
nálægðar
er
óheimil,
svo
sem
menningarstarfsemi,
starfsemi
hárgreiðslustofa,
snyrtistofa,
nuddstofa,
húðflúrstofa,
hundasnyrtistofa,
sólbaðsstofa
og
önnur
sambærileg
starfsemi.
Takmarkanir
þessar
taka
ekki
til
starfsemi
heilbrigðisstarfsmanna
við
veitingu
heilbrigðisþjónustu
en
í
þeim
tilvikum
er
skylt
að
notast
við
andlitsgrímur.
Ákvæði
þetta
tekur
ekki
til
innanlandsflugs
og
-ferja,
leigubifreiða,
hópbifreiða
og
almenningssamgangna.
</p>

<p>
Skemmtistöðum
og
krám,
sbr.
18.
gr.
reglugerðar
nr.
1277/2016,
um
veitingastaði,
gististaði
og
skemmtanahald,
og
spilasölum
skal
lokað
frá
gildistöku
reglugerðarinnar.
Þá
skulu
aðrir
veitingastaðir
þar
sem
heimilaðar
eru
áfengisveitingar
ekki
hafa
opið
lengur
en
til
kl.
21.00
alla
daga
vikunnar
og
fylgja
ákvæðum
3.
og
4.
gr.
Þó
er
heimilt
að
selja
mat
út
úr
húsi
til
kl.
23.00.
</p>

<p>
Í
þeim
tilvikum
þar
sem
fleiri
tegundir
veitingastaða
eru
skráðar
í
rekstrarleyfi
er
heimilt
að
starfa
áfram
í
samræmi
við
aðrar
tegundir
en
kveðið
er
á
um
í
2.
mgr.
með
þeim
takmörkunum
sem
kveðið
er
á
um
í
reglugerð
þessari.
</p>

<p>
Ekki
er
heimilt
að
veita
tímabundið
leyfi
fyrir
skemmtunum,
svo
sem
tónleikum,
dansleikjum,
brennum
eða
öðrum
viðburðum
sem
ætla
má
að
dragi
að
sér
hóp
fólks
eftir
kl.
21.00.
</p>

<p>
Sund-
og
baðstöðum
sem
og
líkamsræktarstöðvum
skal
lokað.
</p>

<p>
Íþróttir,
þar
með
talið
æfingar
og
keppnir,
barna
og
fullorðinna
hvort
sem
er
innan-
eða
utandyra,
með
eða
án
snertingar
eru
óheimilar.
Einstaklingsbundnar
æfingar
án
snertingar
eru
heimilar,
svo
sem
útihlaup
og
sambærileg
hreyfing.
</p>

<p>
Sviðslistir,
og
sambærileg
starfsemi,
eru
óheimilar.
</p>

<h3 class="article__title">
6.
gr.
<em class="article__name">Hjúkrunarheimili
og
aðrar
heilbrigðisstofnanir.</em>
</h3>

<p>
Heilbrigðisstofnanir,
hjúkrunarheimili
og
aðrar
sambærilegar
stofnanir
skulu
setja
sér
reglur
um
starfsemi
sína,
svo
sem
varðandi
heimsóknir
utanaðkomandi
að
heimilunum
og
stofnununum.
</p>

<h3 class="article__title">
7.
gr.
<em class="article__name">Þrif
og
sótthreinsun
almenningsrýma.</em>
</h3>

<p>
Í
öllum
verslunum,
opinberum
byggingum
og
á
öðrum
stöðum
innandyra
þar
sem
umgangur
fólks
er
nokkur
skal
þrífa
eins
oft
og
unnt
er,
sérstaklega
algenga
snertifleti.
Auk
þess
skal
minna
almenning
og
starfsmenn
á
einstaklingsbundnar
sóttvarnir,
svo
sem
munnlega,
með
merkingum
eða
skiltum.
</p>

<p>
Við
alla
innganga
skal
tryggja
aðgang
að
sótthreinsandi
vökva
fyrir
hendur
og
eins
víða
um
rými
og
talin
er
þörf
á,
meðal
annars
við
afgreiðslukassa
og
innkaupakerrur
í
verslunum.
</p>

<h3 class="article__title">
8.
gr.
<em class="article__name">Undanþágur.</em>
</h3>

<p>
Ráðherra
getur
veitt
undanþágu
frá
takmörkun
á
samkomum
vegna
samfélagslega
ómissandi
innviða
sem
mega
ekki
stöðvast,
svo
sem
á
sviði
raforku,
fjarskipta,
samgangna,
sorphirðu,
heilbrigðisstarfsemi,
félagsþjónustu,
löggæslu,
sjúkraflutninga,
hjálparliðs
almannavarna
eða
slökkviliða.
</p>

<p>
Hið
sama
á
við
vegna
kerfislega
og
efnahagslega
mikilvægrar
starfsemi,
svo
sem
til
að
tryggja
fæðuöryggi,
dreifingu
nauðsynjavara
eða
efnahagslegt
öryggi
ríkisins
með
útflutningi.
Enn
fremur
getur
ráðherra
veitt
undanþágu
vegna
starfa
Alþingis,
dómstóla
og
stjórnsýslu.
</p>

<p>
Ráðherra
getur
veitt
undanþágu
frá
banni
við
íþróttastarfi
fyrir
einstaka
viðburði,
svo
sem
vegna
alþjóðlegra
keppnisleikja
hér
á
landi,
enda
sé
gætt
fyllstu
sóttvarnaráðstafana
að
öðru
leyti.
</p>

<p>
Við
veitingu
undanþága
skal
leitast
við
að
grípa
til
annarra
aðgerða
þannig
að
gætt
sé
að
smitvörnum
eins
og
kostur
er.
</p>

<h3 class="article__title">
9.
gr.
<em class="article__name">Gildistaka.</em>
</h3>

<p>
Reglugerð
þessi,
sem
sett
er
með
heimild
í
12.
gr.
sóttvarnalaga
nr.
19/1997,
tekur
gildi
eftir
því
sem
mælt
er
fyrir
um
í
2.
gr.
Frá
sama
tíma
fellur
úr
gildi
reglugerð
nr.
1015/2020,
um
takmörkun
á
samkomum
vegna
farsóttar.
</p>

<p class="Dags" align="center">
<em>Heilbrigðisráðuneytinu,
30.
október
2020.</em>
</p>

<p class="FHUndirskr" align="center">
F.
h.
r.
</p>

<p align="center">
<strong>Ásta
Valdimarsdóttir.</strong>
</p>

<p align="right">
<em>Ásthildur
Knútsdóttir.</em>
</p>
